omod/src/test/java/org/openmrs/module/owa/web/controller/InstallAppControllerTest.java

```diff
@@ -0,0 +1,73 @@
+package org.openmrs.module.owa.web.controller;
+
+import org.apache.struts.mock.MockHttpServletResponse;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+import org.openmrs.GlobalProperty;
+import org.openmrs.api.context.Context;
+import org.openmrs.api.context.ServiceContext;
+import org.openmrs.messagesource.MessageSourceService;
+import org.openmrs.module.owa.App;
+import org.openmrs.module.owa.AppManager;
+import org.openmrs.web.WebConstants;
+import org.openmrs.web.test.BaseModuleWebContextSensitiveTest;
+import org.springframework.mock.web.MockHttpServletRequest;
+import org.springframework.mock.web.MockServletContext;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.List;
+
+public class InstallAppControllerTest extends BaseModuleWebContextSensitiveTest {
+	
+	MessageSourceService originalMessageSourceService;
+	
```

Comment:
What is the value of this class since i already see the same exact tests in OwaRestControllerTest.java?